alma/lib/API/PaymentValidation.php

```diff
@@ -209,6 +209,19 @@ public function validatePayment($almaPaymentId)
 
             // Update payment's order reference
             $order = $this->getOrderByCartId((int) $cart->id);
+            $custom_data = $payment->custom_data;
+            $custom_data['id_order'] = $order->id;
+
+            try {
+                $alma->payments->edit($payment->id, [
```

Comment:
it is not possible to add it in `order.merchant_reference` ?

Reply:
No, it's for the marchants can export the order_id

Reply:
I have right to use the same try{} for two call api ?

Reply:
nop, the `try catch` must concern one call so, if the first call fail, second one can be done anyway. Then, the log message will be separated and more usefull to debug where error was thrown